src/ng/parse.js

```diff
@@ -1805,6 +1806,27 @@ function $ParseProvider() {
       }
     }
 
+    function $$getAst(exp) {
+      var lexer, ast;
+
+      if (typeof exp !== 'string') {
+        exp = '';
+      }
+      if (isOneTimeBinding(exp)) {
+        exp = exp.substring(2);
+      }
+      lexer = new Lexer($parseOptions);
+      ast = new AST(lexer, $parseOptions);
+      return ast.ast(exp);
+    }
+
+    function isOneTimeBinding(exp) {
+      if (typeof exp !== 'string') {
```

Comment:
We don't need this check afaict.

Reply:
I've added this check to future-proof this utility function, just in case we wanted to use it in contexts where it is not safe to assume that the caller will pass it a string.

If we can assume that it'll always be called with a string then I'll remove the check.